internal/core/common_config/main.go

```diff
@@ -117,17 +118,18 @@ func Main(ctx context.Context, cancel context.CancelFunc) {
 		os.Exit(1)
 	}
 
-	lc.Infof("configuration exists: %v", hasConfig)
+	lc.Infof("Common configuration exists in Config Provider is %v and overwrite flag is %v", hasConfig, f.OverwriteConfig())
```

Comment:
can we clarify this log statement? It is a bit confusing on first read - I needed to look at the code to understand what we are logging here...